### PR TITLE
Update migration path: use the spago-legacy package

### PR DESCRIPTION
**Pull request by f-f**
See #1089

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,20 +191,15 @@ You'll need to use [spago-legacy] for this.
...
 
 # Ready to remove the dhall files and move to the new spago
-npm install spago@next
+npm install -g spago@next
 rm spago.dhall packages.dhall
 ```
 
-> [!NOTE]\
-> Both `spago-legacy` and `spago` use the same NPM package name `spago`. The difference is their version numbers. `spago-legacy` stops at `spago@0.21.0` whereas `spago` is `spago@0.93.X`.
-> If `spago-legacy` is installed globally, `spago` can be installed locally via `npm i spago@next` and then used by prefixing `spago` commands with `npx` (e.g. `npx spago build`). Vice versa also works.
```
[the remark this answers is not in the document]
Comment (f-f) [reply README.md:206]: Sure, but the reason it was there was to clarify how things worked in the migration process and why the two things shared a npm package. Now that the two have separate packages, I feel we don't need to note this anymore?
We could still have a similar note to clarify that the `spago-legacy` package is just `spago@0.21.0` (no other releases are present)

[the remark this answers is not in the document]
Comment (JordanMartinez) [reply README.md:206]: That works for me.

File: README.md [hunk 2]
```diff
@@ -191,20 +191,15 @@ You'll need to use [spago-legacy] for this.
...
+npm install -g spago-legacy
 # You can then create a `spago.yaml` file with `migrate`
-spago migrate
+spago-legacy migrate
 
 # Ready to remove the dhall files and move to the new spago
-npm install spago@next
+npm install -g spago@next
 rm spago.dhall packages.dhall
 ```
```
Comment (f-f) [inline README.md:202]:
```suggestion
> [!NOTE]\
> The `spago-legacy` NPM package has exactly the same content as the `spago@0.21.0` release, which is the last one released from the legacy codebase, and the only release to contain the `migrate` command. The new codebase's releases start from `spago@0.93.0`.
```

Comment (JordanMartinez) [reply README.md:202]: Yeah, that sounds good.

**Pull request merged**